packages/typo3-docs-theme/src/Directives/SiteSetSettingsDirective.php

```diff
@@ -148,29 +143,20 @@ public function loadFileFromDocumentation(BlockContext $blockContext, string $fi
             // and is then passed to absoluteRelativePath() which will set $path = "/Configuration/Sets/File.yaml",
             // but ensure no "../../../" or other path traversal is allowed.
             $path = $parserContext->absoluteRelativePath(str_replace('PROJECT:', '', $filename));
-
-            // Get the current origin Path, usually "/project/Documentation/", and go one level up.
-            $newOriginPath = dirname($pathPrefix) . '/';
-
-            // Temporarily change the path prefix now to "/project/"
-            $adapter->setPathPrefix($newOriginPath);
+            $origin = FlySystemAdapter::createForPath(getcwd());
```

Comment:
This is a somewhat nasty hack. We need to introduce a better way of doing this, but as most people are using the docker setup, this is safe for now.